README.md

```diff
@@ -345,17 +345,16 @@ These may be set using Powershell (**use either `SslCertificate` _or_ `CertFile`
 > reg ADD HKLM\Software\CyberArk\Conjur /v SslCertificate /t REG_SZ /d "-----BEGIN CERTIFICATE-----..."
 ```
 
-Or using a `.reg` registry file (**use either `SslCertificate` _or_ `CertFile` but not both**):
+Or using a `.reg` registry file (**`SslCertificate` value cannot be set this way - you must
+set this value via command line**):
 ```reg
 Windows Registry Editor Version 5.00
 
 [HKEY_LOCAL_MACHINE\SOFTWARE\CyberArk\Conjur]
 "ApplianceUrl"="https://conjur.mycompany.com"
 "Version"=dword:00000005
 "Account"="myorg"
-"SslCertificate"="-----BEGIN CERTIFICATE-----
-...
------END CERTIFICATE-----"
+"CertFile"="C:\path\to\ca.pem"
```

Comment:
should we add a commented line like
```
//"SslCertificate"=file("C:\path\to\ca.pem")
```
too?

Also, sidenote, but is this the only place using `CertFile` is documented?

Reply:
> should we add a commented line like

No - this is a registry file example not a puppet manifest example file :(

> but is this the only place using CertFile is documented?

Correct but we need that to be added back with #105 so I was trying to do a bit of 2 birds w/ one stone

Reply:
With the way this is currently drafted, we say "SslCertificate can't be set in .reg - see example" and then only give an example for setting CertFile (and it's the only one in the docs)

I'd rather have an example of how to set SslCertificate, or add a commit that documents CertFile and includes this change and note that we don't support SslCertificate here (is that true?)

Reply:
> we say "SslCertificate can't be set in .reg - see example"

It's more along the lines of "there's an alternate way to set some of the config info but you can't set SslCertificate with it". The examples for doing it over the CLI are right above this section as the main way to do it. Does that change your thought on what you want to see here?

Reply:
I think to make this most clear we should:
- Add `CertFile` to the table above, and be clear that it should be set to the absolute path where the cert can be found
- Update the table so that the `SslCertificate` entry is clear that it's the _contents_ of the cert file

then I think this is pretty clear.